packages/xstate-wallet/src/logger.ts

```diff
@@ -1,8 +1,9 @@
 import pino from 'pino';
-import {LOG_DESTINATION, ADD_LOGS} from './constants';
 
-// TODO: Is there a better way to determine if we're in a browser context?
-const IS_BROWSER_CONTEXT = process.env.NODE_ENV !== 'test';
+import {LOG_DESTINATION, ADD_LOGS, NODE_ENV} from './config';
+
```

Comment:
By removing this comment, do you mean to say that you think this is the best way to determine if we're in a browser context?

Reply:
No, I'll add it back